packages/modals/src/illustration_modal/index.module.scss

```diff
@@ -0,0 +1,15 @@
+@import '~nebenan-ui-kit';
+
+.root {
+  > header {
+    border-bottom: $border-dark;
+  }
+
+  img {
```

Comment:
can we use `> img` or a class name here?

Reply:
I wouldn't touch this right now. We need to change it and then check every modal. Takes a lot of time.
It's already in favro ticket. I Will fix it if there is time left.